source/Attenuator/WandSettings.h

```diff
@@ -162,8 +162,9 @@ const char WAND_SETTINGS_page[] PROGMEM = R"=====(
       <b>LED Count:</b>
       <select id="ledWandCount" name="ledWandCount">
         <option value="0">5 - Stock</option>
-        <option value="1">29 - LED Strip</option>
+        <!--<option value="1">29 - LED Strip</option>-->
         <option value="2">48 - Frutto</option>
```

Comment:
The option for 48 still needs to be commented out, and the 60 is likely not going to be usable in the near future, if at all.

Reply:
Would it be best to just remove all references to 60 altogether then?

Reply:
Just please leave 29 and 48. We don't yet know if a new version of the 48 will be coming, or if the LED count will change. I've been testing with 29 just to play around with timings and animations, but more importantly experimenting with heat dissipation methods. The biggest issue seen so far is that increasing the LEDs density increases power consumption, and that in turn becomes significant heat. I had been testing materials to see if there's something that can be safely used to mitigate that effect.